app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,15 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.pagination .page-link {
+  --bs-pagination-hover-bg: none;
+  --bs-pagination-disabled-bg: none;
+  --bs-pagination-padding-x: .25rem;
+  --bs-pagination-active-bg: none;
+  --bs-pagination-border-width: 0px;
```

Comment:
When it's zero, units can be omitted.

Reply:
If I remove that, the links will appear like 
<img width="361" alt="Screenshot 2024-08-21 at 10 22 57 AM" src="https://github.com/user-attachments/assets/691d85f5-34bc-4bd7-a5b7-bfc37278aa4a">

instead of 
<img width="241" alt="Screenshot 2024-08-21 at 10 23 09 AM" src="https://github.com/user-attachments/assets/2cfc9a79-fdc8-4675-a3a7-3f3e7069da7e">

Reply:
Can you show me? I'm not seeing that behavior when I try it.

Reply:
<img width="964" alt="Screenshot 2024-08-21 at 9 33 30 AM" src="https://github.com/user-attachments/assets/4457fdd0-685d-4585-a9e3-12be776faf5d">

Reply:
Sorry my coffee hadn't kicked in. I thought you meant to get rid of the line. Nevermind. updated.